src/trackingEvents.d.ts

```diff
@@ -39,17 +46,22 @@ export type TrackingEvents = {
   /** enable renderer */
   enable_renderer: {}
   /** Send kernel, website and renderer versions */
-  initialize_versions: Record<string, string>,
+  initialize_versions: Record<string, string>
   /** when desktop client was detected */
   desktop_launched: {}
 
   ab_test: {
-    experimentId: string,
+    experimentId: string
     variant: string
   }
+
+  /** easily search feature flags and/or variants and it's better for data team */
+  feature_flags: {
+    featureFlags: string[]
+  }
```

Comment:
If it's just an array, isn't it better to send the whole array as is? Is there a requirement to send it as a property of an object?
```suggestion
  feature_flags: string[]
```

Reply:
This if because we need to set the name of the table to be used and the name of the column like 
```
feature_flags -> table
featureFlags -> column (type: string[])
(others keys automatically injected) -> column
```